### ISSUE TITLE
pip package "pyads" not correctly installed (qemux86-64)

### ISSUE THREAD

**Issue by carstenschroeder** [home-assistant/hassio-homeassistant#25]
pyads ist needed for ADS component (https://www.home-assistant.io/components/ads/)

The following error occurs when using this component under HassIO. But it works in a native linux install and also in the non-Hass.io docker image homeassistant/home-assistant (https://hub.docker.com/r/homeassistant/home-assistant)

```
2019-02-15 12:44:21 ERROR (MainThread) [homeassistant.setup] Error during setup of component ads
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/homeassistant/setup.py", line 148, in _async_setup_component
    component.setup, hass, processed_config)  # type: ignore
  File "/usr/local/lib/python3.6/concurrent/futures/thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/usr/local/lib/python3.6/site-packages/homeassistant/components/ads/__init__.py", line 57, in setup
    import pyads
  File "/usr/local/lib/python3.6/site-packages/pyads/__init__.py", line 5, in <module>
    from .ads import open_port, close_port, get_local_address, read_state, \
  File "/usr/local/lib/python3.6/site-packages/pyads/ads.py", line 23, in <module>
    from .pyads_ex import (
  File "/usr/local/lib/python3.6/site-packages/pyads/pyads_ex.py", line 46, in <module>
    _adsDLL = ctypes.CDLL(adslib)
  File "/usr/local/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: Error loading shared library /usr/local/lib/python3.6/site-packages/pyads/adslib.so: Exec format error
```

Here is the corresponding configuration:
```
ads:
  device: '5.2.120.114.1.1'
  ip_address: '192.168.6.29'
  port: 801
```

It seems that adslib.so has not been compiled during pip install in docker build. It has the same time stamp and size as in the pip package itself:
![image](https://user-images.githubusercontent.com/13978143/52869964-1e29b680-3147-11e9-8f29-1a6227d6dbbf.png)

Other users had this problem, too. But they also just found out that only Hass.io has this issue:
https://community.home-assistant.io/t/ads-component-does-not-work-with-hassio/77931

**Comment by carstenschroeder** [comment]
I did a test build with 
```
sudo docker run --rm --privileged -v ~/.docker:/root/.docker -v /var/run/docker.sock:/var/run/docker.sock homeassistant/amd64-builder -r https://github.com/home-assistant/hassio-homeassistant -t generic --test --amd64 --homeassistant 0.87.1 --docker-hub homeassistant
```
but no error message is thrown:
![image](https://user-images.githubusercontent.com/13978143/52872916-f25dff00-314d-11e9-9844-1597060062b9.png)

**Comment by carstenschroeder** [comment]
It's because gcc on Alpine doesn't define `__gnu_linux__` only `__linux`, `__linux__` and `linux`.

**Comment by carstenschroeder** [comment]
fixed with pyads 3.0.7
